src/main/java/com/treasure/hunt/game/GameEngine.java

```diff
@@ -26,10 +29,14 @@
  * @author dorianreineccius
  */
 @Requires(hider = Hider.class, searcher = Searcher.class)
+@Getter
+@Setter
 public class GameEngine {
+
     public static final int HEIGHT = 200;
     public static final int WIDTH = 200;
-
+    @Getter
+    private final Statistic statistics= new Statistic();
```

Comment:
Please put this into GameManager, since it is not used by the GameEngine and
code for tools does not belong into GameEngine

Reply:
No the GameEngine needs to be able to add statistics

Reply:
Depending on the type of GameEngine there can be different statistics

Reply:
Well u can also suggest other ways to do this but putting it in the game manager is a bad one

Reply:
Ok i think, you're right with this